server/util/hart_parse.py

```diff
@@ -0,0 +1,40 @@
+from typing import Union, Optional
+from collections import defaultdict
+from xml.etree import ElementTree as ET
+
+NAMESPACE = "http://tempuri.org/CVRDesign.xsd"
+
+
+def find_text_xml(xml: Optional[Union[ET.ElementTree, ET.Element]], tag: str):
+    if xml is None:
+        return None
+    result = find_xml(xml, tag)
+    return None if result is None else result.text
```

Comment:
Is the text ever an empty string? If so, should we convert to `None`?

Reply:
Not in my testing but that is probably a good idea